src/components/screens/secondPassphrase/summaryStep.js

```diff
@@ -22,6 +21,8 @@ const SummaryStep = ({
           passphrase: account.passphrase,
           account: account.info.LSK,
           callback: ({ success, error }) => {
+            // TODO: To be tackled during multiStep refactoring
+            /* istanbul ignore next */
```

Comment:
AFAIK this is just because of the skipped test case for failure path. Failure path should IMO be part of this PR, and then it should be fine to remove this comment.

Reply:
I have implemented the failure path but I still need design to provide the final copy and to let me know if there is anything else there that would need to change in case of failure so I will get back to this when I hear back from them.

Reply:
✅